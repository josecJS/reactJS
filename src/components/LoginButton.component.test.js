import LoginButton from './LoginButton.component';
import renderer from 'react-test-renderer';

describe("Componete botÃ³n", () => {
  test("Es igual a la snapshot del componente", () => {
    const button = renderer.create(<LoginButton />);
    expect(button.toJSON()).toMatchSnapshot();
  });
});