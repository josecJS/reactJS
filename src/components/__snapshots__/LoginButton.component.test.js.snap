// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componete bot√≥n Es igual a la snapshot del componente 1`] = `
<button>
  Conectarse
</button>
`;
